config/queue.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+return [
+
+    /*
+    |--------------------------------------------------------------------------
+    | Default Queue Connection Name
+    |--------------------------------------------------------------------------
+    |
+    | Laravel's queue API supports an assortment of back-ends via a single
+    | API, giving you convenient access to each back-end using the same
+    | syntax for every one. Here you may define a default connection.
+    |
+    */
+
+    'default' => env('QUEUE_CONNECTION', 'sync'),
```

Comment:
Is the default set as `sync` because of the scheduled jobs?

Reply:
Well, it's the default, didn't change it. Should it be `redis_low` or `redis_high`?

Reply:
Just `redis`.